src/main/scala/com/lynbrookrobotics/sbt/Tasks.scala

```diff
@@ -218,6 +220,33 @@ object Tasks {
     }
   }
 
+  lazy val robotConsole: Def.Initialize[Task[Unit]] = Def.task {
+
+    val SIZE = 1024
```

Comment:
This appears to be indented at a different level than the other code. Can you reformat?

Reply:
Can you elaborate? I don't understand what you are saying. When I reformat on IntelliJ nothing changes.